### DISCUSSION TITLE
Running Fabric returns "Failed to fetch models: HTTP 401"

### DISCUSSION THREAD

**Discussion by joelontheroad** [danielmiessler/Fabric#172] [Q&A]
Now that I have fabric correctly installed, whenever I run a fabric command, I get this error:

`Failed to fetch models: HTTP 401`

This occurs with each command, such as:

```
fabric --help
fabric --listmodels
```
etc.

I there authentication with fabric other than the API keys? 

What should I do to resolve this?

**Comment by xssdoctor** [comment]
Make sure the api key is correctly entered. To check, look at ~/.config/fabric/.env. That error message means that your openai api key isn’t entered correctly

**Comment by joelontheroad** [comment]
@xssdoctor you were right.  The API keys were both on one line without spaces.  I put them on their own line and it all worked when I restarted my terminal.

Thank you.
